applications/GeoMechanicsApplication/custom_elements/transient_thermal_element.h

```diff
@@ -242,13 +242,17 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) TransientThermalElement : public Ele
 
     Vector CalculateIntegrationCoefficients(const Vector& rDetJContainer) const
     {
+        const auto& r_properties = GetProperties();
         const auto& r_integration_points = GetGeometry().IntegrationPoints(GetIntegrationMethod());
 
         auto result = Vector{r_integration_points.size()};
         for (unsigned int integration_point_index = 0;
              integration_point_index < r_integration_points.size(); ++integration_point_index) {
             result[integration_point_index] = r_integration_points[integration_point_index].Weight() *
                                               rDetJContainer[integration_point_index];
+            if (GetGeometry().LocalSpaceDimension() == 1) {
+                result[integration_point_index] *= r_properties[CROSS_AREA];
+            }
```

Comment:
Would this be the same for the Pw variant? If yes, do you think it's best to fix that in the same PR, or a different one?

Reply:
@rfaasse Thank you for the review.
Indeed for Pw a similar feature needs to be added. However, currently I am focousing on the heat only to reach the sprint goal. Will do it for Pw is a seperate PR